RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -697,3 +697,120 @@ boolean LX_meatMaid()
 
 	return false;
 }
+
+string LX_getDesiredWorkshed(){

```

Comment:
Change from returning `string` to `item`. That way you don't have to convert to an item later